python/lsst/obs/base/yamlCamera.py

```diff
@@ -96,7 +96,11 @@ def makeDetectorConfigList(ccdParams):
         transforms = ccd["transformDict"]["transforms"]
         detectorConfig.transformDict.transforms = None if transforms == "None" else transforms
         detectorConfig.refpos_x, detectorConfig.refpos_y = ccd["refpos"]
-        detectorConfig.offset_x, detectorConfig.offset_y = ccd["offset"]
+        if len(ccd['offset']) == 2:
```

Comment:
Is x,y deprecated or a fully supported feature of yaml camera? I ask because you seem to have edited all the YAML camera files to add the 0.0 so that suggests you would like to move towards x,y,z being required (otherwise why make the edits at all). If your plan is deprecation then you need to issue a deprecation warning here (but only the first time in the loop).

Reply:
Still deciding, frankly.  But good call on the deprecation warning if we decide to go that way.

Reply:
Update: I've decided that x,y should continue to be supported (pending review I guess).  So no deprecation warning for now.

Reply:
(And also went back and _un_-updated all the camera yaml files.